README.md

```diff
@@ -19,98 +19,125 @@ npm install mosip/tuvali#v0.3.7
+Firstly, for establishing the secured connection over BLE the Verifier's URI needs to be exchanged between two devices. The exchange of URI can be accomplished, but is not limited to, by using a QR code.
 
-For example use QR code generator to visually display params and QR code scanner to get params. A mobile app that displays a QR code can act as an `advertiser` by including its connection params as data in the QR code and another device can act as `discoverer` which scans the QR code, it can extract the connection params and initiate a BLE connection with the advertising device.
+For example use QR code generator to visually display URI and QR code scanner to read. A mobile app that displays a QR code can act as an `Verifier` by including its URI as data in the QR code and another device can act as `Wallet` which scans the QR code, it can extract the URI and initiate a BLE connection with the advertising device.
 
-> Note: The terms `advertiser` and `discoverer` are used for legacy API compatibility reasons. These terms will be replaced with `Verifier` and `Wallet` soon. Hence the terms are used interchangeably as required.
+## URI exchange and Establishing connection
 
-## Connection parameters exchange
-The device on which the QR code is displayed shall generate connection parameters using getConnectionParameters() method:
+### Verifier
+The Verifier device will be showing a QR code with URI. Verifier can generate URI and start advertisement using startAdvertisement() method. Once advertisement started, `Verifier` will keep on advertising URI and will wait for the `Wallet` to connect.
 
 ```typescript
 import OpenIdBle from 'react-native-openid4vp-ble';
-const { Openid4vpBle } = OpenIdBle;
+const { verifier } = OpenIdBle;
 
-const params = Openid4vpBle.getConnectionParameters();
-console.log(params);
+const uri = verifier.startAdvertisement();
+console.log(uri);
 ```
 
-The device that scans the QR code will extract the connection parameters from QR code and set its connection parameters using setConnectionParameters() method :
+The URI contains:
 
-```typescript
-Openid4vpBle.setConnectionParameters(params);
 ```
-The connection params contains:
-
-```json
-{
-"cid": "ilB8l",
-"pk": "4f56504d4f5349505f66b067c008A4484AEC5A769CED2307F59E43DC81A3F768"
-}
+OPENID4VP://4f56504d4f5349505f66b067c008A4484AEC5A769CED2307F59E43DC81A3F768
 ```
-The `pk` part of the data is the same data that will be advertised by the `advertiser` device but in hex encoded form. 
-E.g: OVPMOSIP_<first 5 bytes of public key>
 
-Note: The "cid" parameter is there for legacy reasons and will be removed soon.
+`OPENID4VP` is used as identifier for URI. The part after the `//` of the data is the same data that will be advertised by the `Verifier` device but in hex encoded form.
```

Comment:
Remove this line and point to this link for URI format -> https://bitbucket.org/openid/connect/src/master/openid-4-verifiable-presentations-over-ble/openid-4-verifiable-presentations-over-ble-1_0.md

Reply:
updated